src/Illuminate/Mail/Transport/MandrillTransport.php

```diff
@@ -45,7 +45,7 @@ public function send(Swift_Mime_Message $message, &$failedRecipients = null)
             'key' => $this->key,
             'to' => $this->getToAddresses($message),
             'raw_message' => $message->toString(),
-            'async' => false,
+            'async' => true,
```

Comment:
Yeh, that'll be so people could read the recipient information after send, but that PR should never have been merged, because it prevents mandrill's policy of queuing up messages to be sent in the next hour if someone's over the limit, so introduces an actual bug. Now that we're not letting people access the response, this change should be fine.